utilities/tools/Test-NamePrefixAvailability.ps1

```diff
@@ -34,6 +38,9 @@ function Test-NamePrefixAvailability {
         [Parameter(Mandatory = $true)]
         [string] $namePrefix,
 
+        [Parameter(Mandatory = $true)]
+        [string] $serviceShort,
```

Comment:
This parameter will become a problem as you won't have all 400 serviceShorts at hand when running the script ;)

Ideally, in the logic below, you should simply fetch the serviceShort from the file that you're anyways processing and test against the full name